tox.ini

```diff
@@ -0,0 +1,57 @@
+[tox]
+requires =
+	tox>=4
+env_list = lint, integration, integration-shared-db, integration-database
+
+[testenv]
+setenv =
+	version=8.0
+	repo=https://github.com/canonical/mysql-router-operator.git
+	base-name=docker.io/dataplatformoci/mysql-router
+
+[testenv:lint]
+description = run linters
+skip_install = true
+deps =
+	yamllint
+commands = 
+	yamllint --no-warnings rockcraft.yaml .github
+
+[testenv:integration]
```

Comment:
i believe that the `integration` tox target in `mysql-router-k8s-operator` should be deprecated (as it is intended to run all the integration tests in the repository).

thoughts on finding a way to run all "integration test" target in the `mysql-router-k8s-operator` (so they dont need to be explicitly specified anytime a new suite of tests are added to the repository)?

Reply:
If the plan is to deprecate it, I can reconfigure the tox environment here and the CI to reflect that.

I'll update the PR.

Reply:
i double checked, fairly confident that the `integration` tox target should be deprecated - it runs all the integrations tests in the same model (which is likely to fail as the tests are designed to run in a model per test file)